src/calliope/model.py

```diff
@@ -561,8 +526,8 @@ def validate_math_strings(self, math_dict: dict) -> None:
         """
```

Comment:
I believe this function belongs here and not in `ModelMath` since it is aware of the existence of `_model_data` and `math`.

Reply:
Hmmm...
I think it's fine here. The math and parser classes are specialized, and this function is aware of the overall structure of calliope. I will change the validation to use the math object's method, though. I forgot to switch it...